src/management/OrganizationsManager.js

```diff
@@ -0,0 +1,816 @@
+ *
+ * @method    createInvite
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params =  { id :'ORGANIZATION_ID'};
+ * var data = {}
+ *
+ * management.organizations.createInvite(params, data, function (err) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ * });
+ *
+ * @param   {String}    params.id             ID of the Organization.
+ * @param   {Array}     data                  Invitation data
+ * @param   {Function}  [cb]                  Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+OrganizationsManager.prototype.createInvite = function(params, data, cb) {
+  data = data || [];
+  params = params || {};
+
+  if (!params.id) {
+    throw new ArgumentError('The organization ID passed in params cannot be null or undefined');
+  }
+  if (typeof params.id !== 'string') {
+    throw new ArgumentError('The organization ID has to be a string');
+  }
+
+  if (cb && cb instanceof Function) {
+    return this.invitations.create(params, data, cb);
+  }
+
+  return this.invitations.create(params, data);
+};
+
+/**
+ * Delete an invitation from an organization
+ *
+ * @method    deleteInvitation
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params =  { id :'ORGANIZATION_ID', invitation_id: 'INVITATION_ID };
+ *
+ * management.organizations.deleteInvitation(params, function (err) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ * });
+ *
+ * @param   {String}    params.id             ID of the Organization.
+ * @param   {String}    params.invitation_id  Invitation ID
+ * @param   {Function}  [cb]                  Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+OrganizationsManager.prototype.deleteInvite = function(params, cb) {
+  params = params || {};
+
+  if (!params.id) {
+    throw new ArgumentError('The organization ID passed in params cannot be null or undefined');
+  }
+  if (typeof params.id !== 'string') {
+    throw new ArgumentError('The organization ID has to be a string');
+  }
+
+  if (!params.invitation_id) {
+    throw new ArgumentError('The invitation ID passed in params cannot be null or undefined');
+  }
+  if (typeof params.invitation_id !== 'string') {
+    throw new ArgumentError('The invitation ID has to be a string');
+  }
+
+  if (cb && cb instanceof Function) {
+    return this.invitations.delete(params, {}, cb);
+  }
+
+  return this.invitations.delete(params, {});
+};
+
+/**
+ **** Organization Roles Membership
+ */
+
+/**
+ * Get Roles from a Member in a Organization
+ *
+ * @method    getRoles
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params = {id : 'ORGANIZATION_ID', user_id: 'user_id'}
+ * @example <caption>
+ *   This methods takes the organization ID and user ID and returns the roles
+ * </caption>
+ *
+ * management.organizations.getRoles( {id : 'ORGANIZATION_ID', user_id: 'user_id'}, function (err, invite) {
```

Comment:
Also shouldn't this match to the function name? (`getMemberRoles`?)

Reply:
Also `function (err, invite) {` seems like a copy paste error, should be `roles` or something.